src/app/account/user-details/user-details.component.html

```diff
@@ -160,9 +156,6 @@ <h2>
       </form>
     </div>
   </div>
-  <div *ngIf="detailsOption === 6" class="details-address">
```

Comment:
Ce zici sa declaram un Enum cu DetailsOption.
Un enum arata asa: 
```
enum DetailsOption {
Address = 2,
RecommendedBikes = 6
}
```
Si in HTML ar arata asa:
```<div *ngIf="detailsOptions === DetailsOptions.RecommendedBikes"```
Mai multe despre enums: https://www.typescriptlang.org/docs/handbook/enums.html
Folosind enums  e mai usor sa citesti cod-ul si pe termen lung, apar mai putine bug-uri.